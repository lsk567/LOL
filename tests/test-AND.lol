bool h = true;
bool i = true;
bool j = false;

bool t1 = h AND i;
bool t2 = h AND j;

if (t1) {
	println("T AND T is T");
} else {
	println("T AND T is F");
}

if (t2) {
	println("T AND F is T");
} else {
	println("T AND F is F");
}


