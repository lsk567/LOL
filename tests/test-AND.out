T AND T is T
T AND F is F
